### ISSUE TITLE
NVDA does not return the sound of the audios to normalized using the voice synthesizer windows one core

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7431]
I changed the nvda synthesizer to the one-core windows voices and noticed a problem.
When playing a song using any audio player the nvda lowers the sound as long as there is talk, however when the voice for the sound does not return to normalised Audio.
The nvda's hen reduction setting is to redirect sounds when playing speech.
This problem did not occur when using another speech synthesizer as nuance vocalizer exprecive 1.1.
I am with nvda next 14215 and windows 10 15063.483.

**Comment by derekriemer** [comment]
@jcsteh

**Comment by jcsteh** [comment]
Confirmed. STR:

1. Switch to the Windows OneCore Voices synthesiser.
2. Play some music.
3. Use NVDA+shift+d to set NVDA's audio ducking mode to "Duck when outputting speech and sounds".
    - Expected: After NVDA finishes speaking, audio should be unducked after a short delay.
    - Actual: NVDA finishes speaking, but audio is never unducked.

I must be missing a call to WavePlayer.idle somewhere.
